extensions/security-jpa/runtime/src/main/java/io/quarkus/security/jpa/runtime/Password.java

```diff
@@ -0,0 +1,26 @@
+package io.quarkus.security.jpa.runtime;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * <p>
+ * Indicates that this field or property should be used as a source of password for security. Only
+ * supports the {@link String} type.
+ * </p>
+ * <p>
+ * Defaults to considering the password as hashed with bcrypt in the Modular Crypt Format.
+ * </p>
```

Comment:
For the record, Javadoc complains about the closing `</p>`.

Reply:
I can't imagine why.

Reply:
I guess because the end tag is optional.

Reply:
What nonsense.

Reply:
It doesn't help matters that IntelliJ aggressively creates end tags...